src/pointcloud_io.jl

```diff
@@ -354,7 +354,7 @@ function make_table(points::AbstractVector{LasPoint3}, offset, scale)
     classification = map(p -> p.raw_classification, points)
     pointsourceid = map(p -> p.pt_src_id, points)
     gpstime = map(p -> p.gps_time, points)
-    userdata = map(p -> p.user_data, points)
+    # userdata = map(p -> p.user_data, points)
```

Comment:
remove

Reply:
same below old code

Reply:
actually why are you remioving the userdata?

Reply:
That's about the problem I mentioned, If we leave user data we can't write las point 3 anymore, all will be 7.